cocos/core/pipeline/geometry-renderer.ts

```diff
@@ -897,6 +902,25 @@ export class GeometryRenderer {
         }
     }
 
+    public addSpline (spline: Spline, color: Color, index = 0xffffffff, knotSize = 0.5, segments = 32, depthTest = true) {
```

Comment:
Why default value of  index is 0xffffffff? It is a big number.

Reply:
It is -1, declare as an uint32_t in native

Reply:
How about define invalidIndex to replace 0xffffffff?